faunadb/query.py

```diff
@@ -58,8 +49,7 @@ def lambda_query(func):
       #  "expr": {"add": ({"var": "auto0"}, {"var": "auto0"})}
       # }
 
-  Query functions requiring lambdas can be passed Python lambdas
-  without explicitly calling :any:`lambda_query`.
+  You usually don't need to call this directly as lambdas in queries will be converted for you.
   For example: ``query.map(lambda a: query.add(a, 1), collection)``.
```

Comment:
It's meant to be more familiar to python developers; in python you write `map(lambda x: x + 1, [1, 2, 3])`, which is the reverse of ruby/js style.